regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
```

Comment:
You may be right. I had thought I had an example, but it's being handled by `INCLUDE_DEFINITIONS_IN` instead. Going to dig in and make sure this is actually being used and find out what for.